src/contexts/NotificationProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+const NotificationContext = React.createContext({
+  notifications: [],
+  addNotification: (variant: 'success' | 'danger' | 'info', message: string) => {},
```

Comment:
can you change this to be like:
`addNotification: (variant: NotificationVariantType, message: string) => null` 
And for line below change it to 
`removeNotification: (key: string) => null`?

(i.e., change the {} to null at the end).

It's failing the linter as is because of a typescript/eslint/no-empty-arrow-function rule.

Reply:
I had no idea how to get the linter to stop complaining about that.  That seems to work.  It's still mad that I'm not using the args of the function, but it seems less mad about that.